src/commands/translate.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  ApplicationCommandType,
+  EmbedBuilder,
+  Colors,
+  Message,
+} from "discord.js";
+import { createCommand } from "../create-command.ts";
+import OpenAI from "openai";
+import { config } from "../config.ts";
+
+const openai = new OpenAI({ apiKey: config.OPENAPI_KEY });
```

Comment:
It would be cleaner to initialise this once in index.ts and pass it through the context